docs/topics/braintree.rst

```diff
@@ -57,3 +57,46 @@ Braintree already, it is not created.
 
     :status 200: customer and buyer already exist.
     :status 201: customer or buyer successfully created.
+
+Create a payment method
+-----------------------
+
+Creates a payment method in Braintree and the corresponding payment method in
+solitude.
+
+.. http:post:: /braintree/method/
+
+    :<json string uuid: the uuid of the buyer in solitude.
+    :<json string nonce: the payment nonce returned by Braintree.
```

Comment:
I was trying to find something in brainree's docs that we can link to but I can't actually find it now :(